From 86a3fa50a4eda747b18da4844ace253cbb8db4d8 Mon Sep 17 00:00:00 2001
From: Fabrizio Castro <fabrizio.castro.jz@renesas.com>
Date: Mon, 25 Oct 2021 10:17:22 +0100
Subject: [PATCH] arm64: dts: renesas: r9a07g044l: Fix the USB PHY interrupt

The interrupts registered with the USB PHYs are not correct,
fix them.
While at it, adjust the formatting of USB related nodes.

Signed-off-by: Fabrizio Castro <fabrizio.castro.jz@renesas.com>
---
 arch/arm64/boot/dts/renesas/r9a07g044l.dtsi | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/arch/arm64/boot/dts/renesas/r9a07g044l.dtsi b/arch/arm64/boot/dts/renesas/r9a07g044l.dtsi
index ae071ae9a1f4..c382166f2c15 100644
--- a/arch/arm64/boot/dts/renesas/r9a07g044l.dtsi
+++ b/arch/arm64/boot/dts/renesas/r9a07g044l.dtsi
@@ -435,7 +435,7 @@
 			status = "disabled";
 		};
 
-		ohci1: usb@e11c70000 {
+		ohci1: usb@11c70000 {
 			compatible = "generic-ohci";
 			reg = <0 0x11c70000 0 0x100>;
 			interrupts = <GIC_SPI 96 IRQ_TYPE_LEVEL_HIGH>;
@@ -477,7 +477,7 @@
 			compatible = "renesas,usb2-phy-r9a07g044l",
 				     "renesas,rcar-gen3-usb2-phy";
 			reg = <0 0x11c50200 0 0x700>;
-			interrupts = <GIC_SPI 90 IRQ_TYPE_LEVEL_HIGH>;
+			interrupts = <GIC_SPI 94 IRQ_TYPE_LEVEL_HIGH>;
 			clocks = <&cpg CPG_MOD R9A07G044L_CLK_USB0>;
 			power-domains = <&cpg>;
 			resets = <&cpg R9A07G044L_CLK_USB0>;
@@ -485,11 +485,11 @@
 			status = "disabled";
 		};
 
-		usb2_phy1: usb-phy@11C70200 {
+		usb2_phy1: usb-phy@11c70200 {
 			compatible = "renesas,usb2-phy-r9a07g044l",
 				     "renesas,rcar-gen3-usb2-phy";
-			reg = <0 0x11C70200 0 0x700>;
-			interrupts = <GIC_SPI 95 IRQ_TYPE_LEVEL_HIGH>;
+			reg = <0 0x11c70200 0 0x700>;
+			interrupts = <GIC_SPI 99 IRQ_TYPE_LEVEL_HIGH>;
 			clocks = <&cpg CPG_MOD R9A07G044L_CLK_USB1>;
 			power-domains = <&cpg>;
 			resets = <&cpg R9A07G044L_CLK_USB1>;
-- 
2.25.1

